docs/training/apis-howto/api-core-ug.rst

```diff
@@ -0,0 +1,416 @@
+#########################
+ Core API
+#########################
+
+In this guide, you'll learn how to use the Core API.
+
++------------------------------------------------------------------+
+| Visit the API reference                                          |
++==================================================================+
+| :doc:`/reference/reference-training/training/api-core-reference` |
++------------------------------------------------------------------+
```

Comment:
does it make sense to put the API reference link at the end of the overview?

Reply:
it does make sense. i'll keep them at the top for now because they act like a toggle, toggling back and forth between the guide and the corresponding ref doc. i'll add a task for myself to revisit all of our "ascii" tables because I need to reformat them. resolving comment.

https://determinedai.atlassian.net/browse/DOCS-9?atlOrigin=eyJpIjoiZmY4YzU1ZmVmOGZmNDBlNWI5YTQ1YjVmNTY4YTJmZGUiLCJwIjoiaiJ9